cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -66,6 +68,13 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
But [WebView.setWebContentsDebuggingEnabled()](https://developer.android.com/reference/android/webkit/WebView.html#setWebContentsDebuggingEnabled(boolean)) is available since API 19(Android OS 4.4). It will cause compiling error if on Android OS 2.3.